crates/starknet-types-core/Cargo.toml

```diff
@@ -44,6 +46,7 @@ serde = ["alloc", "dep:serde"]
 prime-bigint = ["dep:lazy_static"]
 num-traits = []
 papyrus-serialization = ["std"]
+zeroing = ["zeroize"]
```

Comment:
zeroing = ["dep:zeroize"]
in ordr to disable the implicit feature.
https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

Currently you have two features available 
`zeroing` the one you defined
and 
`zeroize` from the default behaviour of `optional = true`